src/server/components/lists/controllers/index.ts

```diff
@@ -92,11 +121,40 @@ export function listsGetController(req: Request, res: Response): void {
   }
 
   const { serviceType } = params;
+  let { languagesProvided, servicesProvided } = params;
 
   let questionsSequence: QuestionName[];
   let partialPageTitle: string = "";
+  let partialPageHintText: string = "";
   let partialToRender: string = "";
   let error: boolean | QuestionError = false;
+  let partialData: QuestionDataSet[] | QuestionData[];
+  let languagesRows, languageNamesProvided, serviceNamesProvided;
```

Comment:
Code styling wise, it looks like the other variables have been given type annotations but the variables on line **132** don't have any.

I'm not super bothered by this but for consistency it might make sense to give these variables types?

Reply:
fair point.  I've changed this now.